mne/decoding/transformer.py

```diff
@@ -715,3 +719,166 @@ def transform(self, X):
+              the shortest power-of-two length at least that duration for
+              ``phase="zero-double"``.
+    l_trans_bandwidth : float | str
+        Width of the transition band at the low cut-off frequency in Hz
+        (high pass or cutoff 1 in bandpass). Can be "auto"
+        (default in 0.14) to use a multiple of ``l_freq``::
+            min(max(l_freq * 0.25, 2), l_freq)
+        Only used for ``method='fir'``.
+    h_trans_bandwidth : float | str
+        Width of the transition band at the high cut-off frequency in Hz
+        (low pass or cutoff 2 in bandpass). Can be "auto"
+        (default in 0.14) to use a multiple of ``h_freq``::
+            min(max(h_freq * 0.25, 2.), info['sfreq'] / 2. - h_freq)
+        Only used for ``method='fir'``.
+    n_jobs : int | str
+        Number of jobs to run in parallel. Can be 'cuda' if scikits.cuda
+        is installed properly, CUDA is initialized, and method='fft'.
+    method : str
+        'fir' will use overlap-add FIR filtering, 'iir' will use IIR
+        forward-backward filtering (via filtfilt).
+    iir_params : dict | None
+        Dictionary of parameters to use for IIR filtering.
+        See mne.filter.construct_iir_filter for details. If iir_params
+        is None and method="iir", 4th order Butterworth will be used.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+        Defaults to self.verbose.
+
+    See Also
+    --------
+    FilterEstimator
+    Vectorizer
+    """
+    def __init__(self, l_freq=None, h_freq=None, sfreq=1.0,
+                 filter_length='auto', l_trans_bandwidth='auto',
+                 h_trans_bandwidth='auto', n_jobs=1, method='fir',
+                 iir_params=None, verbose=None):
+        self.l_freq = l_freq
+        self.h_freq = h_freq
+        self.filter_length = filter_length
+        self.l_trans_bandwidth = l_trans_bandwidth
+        self.h_trans_bandwidth = h_trans_bandwidth
+        self.n_jobs = n_jobs
+        self.method = method
+        self.iir_params = iir_params
+        self.sfreq = sfreq
+
+        dummy_array = [1, 2]
+
+        (_, self.sfreq, self.l_freq, self.h_freq, self.l_trans_bandwidth,
+         self.h_trans_bandwidth, self.filter_length, _) = \
+            _triage_filter_params(dummy_array, sfreq, l_freq, h_freq,
+                                  l_trans_bandwidth, h_trans_bandwidth,
+                                  filter_length, method, phase='zero')
+
+        if not isinstance(self.n_jobs, int) and self.n_jobs == 'cuda':
+            raise ValueError('n_jobs must be int or "cuda", got %s instead.'
+                             % type(self.n_jobs))
+
+        if self.l_freq == 0:
+            self.l_freq = None
+        if self.h_freq is not None and self.h_freq > (self.sfreq / 2.):
+            self.h_freq = None
+        if self.l_freq is not None and not isinstance(self.l_freq, float):
+            self.l_freq = float(self.l_freq)
```

Comment:
Should not need to do this, the triage function should return `float` or `None` already.